test/crdsvalidation/kongkeyset/testcases/controlplaneref.go

```diff
@@ -138,5 +138,16 @@ var cpRef = testCasesGroup{
 			},
 			ExpectedErrorMessage: lo.ToPtr("spec.controlPlaneRef cannot specify namespace for namespaced resource - it's not supported yet"),
 		},
+		{
+			Name: "no cpRef is valid",
+			KongKeySet: configurationv1alpha1.KongKeySet{
+				ObjectMeta: commonObjectMeta,
+				Spec: configurationv1alpha1.KongKeySetSpec{
+					KongKeySetAPISpec: configurationv1alpha1.KongKeySetAPISpec{
+						Name: "name",
+					},
+				},
+			},
+		},
```

Comment:
Given that we do not plan to support `KongKeySet` in KIC is this use case really valid? 🤔  Ditto for other types that are not supported in KIC.

Reply:
Yeah, I'd remove the check for CRDs that are not supported for KIC. Currently only `KongConsumer`s and `KongConsumerGroups` are supported by KIC.